frontend/packages/pipelines-plugin/integration-tests/support/pages/pipelines/pipelineBuilder-page.ts

```diff
@@ -86,8 +86,9 @@ export const pipelineBuilderPage = {
         detailsPage.titleShouldContain(pageTitle.Pipelines);
         cy.get('body').then(($body) => {
           if ($body.find('[data-test-id= "dropdown-button"]').length > 0) {
-            cy.byLegacyTestID('dropdown-button').click();
-            cy.get(pipelineBuilderPO.pipeline).click();
+            cy.get(pipelinesPO.createPipeline).click();
+            // cy.byLegacyTestID('dropdown-button').click();
+            // cy.get(pipelineBuilderPO.pipeline).click();
```

Comment:
The `Create Pipeline` button becomes a dropdown once pipeline-as-code repositories are available (some of our tests use that). This code was meant to handle that case.